graphql-cli/src/test/resources/serviceGen/graphqlSchemas/valid/SchemaWithInputTypeDefaultParametersApi.graphql

```diff
@@ -0,0 +1,19 @@
+type Query {
+    book(id: Int!, title: String): Book
```

Comment:
How do we handle default values on primitive inputs? 

```graphql
book(id: Int! = 1, title: String): Book
```

Shall we have a test for that too?

Reply:
I have added tests to handle these scenarios in the previous pull request.

Reply:
My bad.